vulnerabilities/api.py

```diff
@@ -182,6 +183,14 @@ def to_representation(self, instance):
 
         weaknesses = data.get("weaknesses", [])
         data["weaknesses"] = [weakness for weakness in weaknesses if weakness is not None]
+        alias_queryset = instance.aliases.all()
+        data["advisory"] = []
+        for itr in alias_queryset:
+            advisory_objects = Advisory.objects.filter(aliases=[itr.alias])
+            for i in advisory_objects:
```

Comment:
please try to use meaningful variable names when iterating (  `i`,  `itr` )

```suggestion
            for advisory_object in advisory_objects:
```